docs/Q-and-A.md

```diff
@@ -0,0 +1,277 @@
+
+On github KERI is - and will become even more - a thickening bunch of repositories:<br/>
+ 1. https://github.com/decentralized-identity/keri 
+ 2. https://github.com/decentralized-identity/keripy
+ 3. etc.
+ Lastly, the important man, who founded KERI is *Samuel M. Smith Ph.D.*, operationing from his firm [prosapien.com](https://www.prosapien.com). 
+
+## In what programming languages is KERI available?
+In Python. It will be available in the coming year in Rust, Javascript and Go (2020).
+
+## How KERI fit in [the 10 principles of SSI](https://medium.com/metadium/self-sovereign-identity-principle-6-portability-4a7105dd0381) by Christopher Allen?
+KERI is not primarily about self-sovereign identity. KERI is primarily about autonomic identifiers, AIDs. That is identifiers that are self managing. KERI provides proof of control authority over the identifier. What one does with the identifier is not constrained by KERI. But because the primary root of trust of an AID is a KEL which can be hosted by any infrastructure, any identity system (SSI or otherwise) built on top of KERI may also be portable. <br/>So in my opnion portability of the associated identifiers is essential to any truly self-sovereign identity system. That portability, as such, is not one of the principles is a defect in the principles.
+
+Christopher Allen is talking about *portability of information* related to the identity, not the *portability of the identifier itself* with respect to its supporting infrastructure (aka spanning layer).  Indeed, most `DID` methods, including those  that publicly claim to be `SSI` in accordance with the principles do not have portable identifiers. They are locked to a given ledger.
+
+# Q&A section KERI operational
+
+## Where can I download KERI?
+On (sub)page(s of) [github](https://github.com/decentralized-identity/keri)
+
+## Where can we find the code and how could a coder get started?
+The homepage on github [README.md](../README.md) pretty much sums up all the possibilities to download the available code and how developers can engage in the development process currently. We welcome all help we can get.
+
+## What would you see as the main drawback of Keri?
+Its main drawback is that it's nascent.
+
+## How can it one solution, fit for all SSI problems? 
+KERI uses plain old digital signatures from `PKI`, intentionally, so that it may be truly universally applied. KERI solves that hard problem of PKI, that is, key rotation in a standard way. Without a standard way of addressing key rotation, there is no interoperability between systems, they break when you rotate keys because no one knows how to verify the key rotation was done properly. `KERI` solves that problem. 
+
+## Where you would need something quite different than KERI?
+`KERI` does one thing, it establishes control authority using verifiable portable proofs that are `KEL`s. 
```

Comment:
I am not sure but this answer does not seem to fit the question. Overall I would change the question to "Where you would need KERI?"

as there is million of places where you need something different then KERI.